### PR TITLE
[RSS-19] Create NavBar Component 

### PR DESCRIPTION
**Pull request by ellabuysse**
Added NavBar component and minimalistic styling.

### CODE REVIEW

File: src/components/NavBar/NavBar.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>
+
+<div class="navbar">
+    <a href="#">Logout</a>
+    <a href="#">Edit Profile</a>
+</div>
+
+</body>
+</html>
```
Comment (jackfales) [inline src/components/NavBar/NavBar.html]: Can you change this to a .jsx file? Check LoginPage.jsx for inspiration. Sorry I didn't specify this earlier, but it should be a quick change.

**Review by jackfales — CHANGES_REQUESTED** [review]
Change .html file to .jsx, otherwise everything looks great! Good job following proper naming conventions for your PR.

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './NavBar.css';
+
+export default () => (
+
+  <div className="navbar">
+    <a href="/">Logout</a>
+    <a href="/">Edit Profile</a>
+  </div>
```
Comment (jackfales) [inline src/components/NavBar/NavBar.jsx:9]:
```suggestion
  return(
  <div className="navbar">
    <a href="/">Logout</a>
    <a href="/">Edit Profile</a>
  </div>
  );
```

File: src/components/NavBar/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './NavBar.css';
+
+export default () => (
```
Comment (jackfales) [inline src/components/NavBar/NavBar.jsx:4]:
```suggestion
export default () => {
```

File: src/components/NavBar/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+  <div className="navbar">
+    <a href="/">Logout</a>
+    <a href="/">Edit Profile</a>
+  </div>
+
+);
```
Comment (jackfales) [inline src/components/NavBar/NavBar.jsx:11]:
```suggestion
};
```

**Review by jackfales — COMMENTED** [review]
Nice work! Just some minor changes so the file is easier to modify in the future and it should be good to go!

**Review by jackfales — APPROVED** [review]

**Comment by ObjectJosh** [comment]
Looks good!

**Pull request merged**